egs2/mediaspeech/asr1/run.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+lid=false # whether to use language id as additional label
```

Comment:
I do not see where you are using language id as an additional label, if not for this corpus, please remove it correspondingly.